tests/test_pipeline.py

```diff
@@ -72,22 +79,26 @@ def test_appsrc_no_caps():
     )
 
     with GstPipeline(cmd) as pipeline:
-        pipeline.set_appsrc_caps(width=320, height=240, framerate=10, format="GRAY8")
+        pipeline.set_appsrc_video_caps(
+            width=320, height=240, framerate=10, format="GRAY8"
+        )
         while pipeline:
             pipeline.push(rand_array(channels=1))
 
 
-def test_appsrc_and_sink():
+def test_video_appsrc_and_sink():
     num_buffers, count = 10, 0
 
     cmd = (
         f"appsrc emit-signals=true num-buffers={num_buffers} ! "
         "appsink emit-signals=true"
     )
     with GstPipeline(cmd) as pipeline:
-        pipeline.set_appsrc_caps(width=320, height=240, framerate=10, format="GRAY8")
+        pipeline.set_appsrc_video_caps(
+            width=320, height=240, framerate=10, format="GRAY8"
+        )
         while pipeline:
-            pipeline.push(rand_array(channels=3))
+            pipeline.push(rand_array(channels=1))
```

Comment:
Wasn't sure if this was an error or this is how the test was designed - The format is meant to be GRAY8 which single channel but the array being pushed has three channels.